CONTRIBUTING.rst

```diff
@@ -121,7 +121,7 @@ Start coding
 .. _pre-commit: https://pre-commit.com
 .. _create a pull request: https://help.github.com/en/articles/creating-a-pull-request
 
-.. _contributing-testsuite:
+.. _contributing-testsuite: https://github.com/pallets/flask/blob/master/CONTRIBUTING.rst#running-the-tests
```

Comment:
Isn't it enough to link to `#running-the-tests`?

Reply:
Yeah, you are right, I have changed it.